test/Generators/Microsoft.Gen.Logging/TestClasses/LogPropertiesSpecialTypesExtensions.cs

```diff
@@ -6,6 +6,8 @@
 using System.Numerics;
 using Microsoft.Extensions.Logging;
 
+#pragma warning disable LOGGEN036
```

Comment:
We shouldn't emit this warning when a parameter is annotated with `[LogProperties]`

Reply:
My understanding was that we emit the warning only when a parameter is presented in message template

Reply:
The warning that gets produced is this:

<img width="793" alt="image" src="https://github.com/dotnet/extensions/assets/22780957/16dff1b8-a237-4732-9553-435e097dcd82">

which I think is correct.

Reply:
I see. Can we maybe move `#pragma` closer to the properties which cause this warning?

Reply:
Done.